src/creature/creature.h

```diff
@@ -21,8 +21,9 @@ class Creature
 		std::string creature_type);
 
 	void PrintStats();
-
 	int get_hp();
+	std::string get_creature_type();
+	int get_strength();
```

Comment:
this getter is super-specific. would you mind having a getter for all of the ability scores, if possible?

Reply:
have you found a better solution for this? it's probably very easy but i just can't make it work

Reply:
Perhaps this answer from this post will help you:
https://stackoverflow.com/questions/3473438/return-array-in-a-function

Basically, treat the ability_scores array like a pointer to its first element. You can return (from the getter function) that pointer, and store it in an appropriate value outside the getter (say, mc_ability_scores). You then access any value you like using normal array syntax, ege mc_ability_scores[0] should get you the strength score. Give it a try :)

Reply:
On a second thought, perhaps this isn't the best idea, exposing the address of a protected member like that. I believe having a getter function for all ability scores that accepts an integer parameter (as large as the number of ability scores) then return the value of that particular ability score, is our best option.

So something along the lines of:
`int get_ability_score(int n)
{
if (n > -1 && n < 6)
{
return m_ability_scores[n];
}
}`